dashboard.py

```diff
@@ -374,27 +348,38 @@ def stop_tracking_product(user_id, product_id):
 def show_main_page():
     """Displays the main page on the dashboard"""
     with mainSection:
+        with st.sidebar:
+            page = option_menu(
+                menu_title="Menu", options=["About", "Current products", "Track new products", "Unsubscribe from product tracking"])
```

Comment:
I think having this sidebar is such a good idea, it makes the dashboard a lot more organised!